features/step_definitions/catalogue_steps.rb

```diff
@@ -41,3 +41,9 @@
 Then(/^I see #{MAYBE_VAR} in the search summary text$/) do |value|
   find(:xpath, "//*[@class='search-summary']/em[1]").text().should == normalize_whitespace(value)
 end
+
+When /^I tick a random filter$/ do
+  checkbox = all(:xpath, "//form[@action='/g-cloud/search']//input[@type='checkbox']").sample
+  page.check(checkbox[:id])
```

Comment:
I've written some general checkbox checking functions in [my currently open pull request](https://github.com/alphagov/digitalmarketplace-functional-tests/pull/259/files).  Currently:
- it will only exist in the `common_steps.rb` file
- the checkboxes in our option-select boxes haven't been updated to show the new styles (not sure if they will be -- but in its current form, this will break if the markup is updated).

[The general checkbox-checking code](https://github.com/alphagov/digitalmarketplace-functional-tests/blob/68470725021d8d57a8f917fab4b64fb3638de82a/features/step_definitions/common_steps.rb#L18-L30) will work with this though, so if we had a way of sharing it between files, that's probable the safest way to go.